graphene_django/forms/tests/test_mutation.py

```diff
@@ -150,6 +183,54 @@ class Mutation(ObjectType):
         self.assertEqual(result.data["myMutation"]["errors"], [])
         self.assertEqual(result.data["myMutation"]["text"], "VALID_INPUT")
 
+    def test_filtering_output_fields_exclude(self):
+        class FormWithWeirdOutput(MyForm):
+            """Weird form that has extra cleaned_data we want to expose"""
+            text = forms.CharField()
+            another = forms.CharField(required=False)
+            def clean(self):
+                super(FormWithWeirdOutput, self).clean()
+                self.cleaned_data["some_integer"] = 5
+                return self.cleaned_data
+
+        class MyMutation(DjangoFormMutation):
+            class Meta:
+                form_class = FormWithWeirdOutput
+                exclude = ["text"]
+
+            some_integer = Int()
+
+        assert "text" in MyMutation.Input._meta.fields
+        assert "another" in MyMutation.Input._meta.fields
+
+        assert "text" not in MyMutation._meta.fields
+        assert "another" in MyMutation._meta.fields
+        assert "some_integer" in MyMutation._meta.fields
+
+        class Mutation(ObjectType):
+            my_mutation = MyMutation.Field()
+
+        schema = Schema(query=MockQuery, mutation=Mutation)
+
+        result = schema.execute(
+            """ mutation MyMutation {
+                myMutation(input: { text: "VALID_INPUT" }) {
```

Comment:
@jkimbo - I am super confused why this test errors with `text not a valid keyword argument` when the assertion statements above are true. Any thoughts?

```
>       self.assertIs(result.errors, None)
E       AssertionError: [GraphQLLocatedError("'text' is an invalid keyword argument for MyMutation")] is not None
```